python/jax/carfac.py

```diff
@@ -2356,11 +2358,14 @@ def run_segment(
   #       (n_ears, cfp.n_ears))
 
   n_ch = hypers.ears[0].car.n_ch
+  n_agc_stages = jnp.shape(state.ears[0].agc)[0]
   naps = jnp.zeros((n_samp, n_ch, n_ears))  # allocate space for result
   naps_fibers = jnp.zeros((n_samp, n_ch, n_fibertypes, n_ears))
+  receptor_pot = jnp.zeros((n_samp, n_ch, n_ears))
```

Comment:
for the receptor pot, is it more of an optional item, a la agc and agc_memory? 

if so, name as such and place at end of outputs.

Reply:
Yup its technically optional and more for debugging + further changes...will move it around